xml/sle_update_upgrading.xml

```diff
@@ -309,8 +309,7 @@
     <term>Upgrading from &productname; 15 SP3</term>
     <listitem>
      <para>
-      Upgrading from &productnameshort;&nbsp;15&nbsp;SP1 or SP2 is supported
-      both online and offline.
+      Upgrading from &productnameshort;&nbsp;15&nbsp;SP3 is supported both online and offline.
```

Comment:
This change looks wrong, let's discuss further.

Reply:
Simple upgrades have always been supported both offline and online. Why would that change?

Reply:
Now reading it in context: This is fine.